src/contexts/PlantContext.tsx

```diff
@@ -128,13 +132,11 @@ export const PlantContextProvider: React.FC<{ children: ReactNode }> = ({
         return true;
     };
 
-    const Lala = ['lalal', 'lala'];
-
     if (!renderChildren()) {
         if (fetchProjectsAndPermissionsStatus === AsyncStatus.ERROR) {
             return (
                 <>
-                    <Navbar />
+                    <Navbar leftContent={<ProcosysButton />} />
```

Comment:
Dette er baren  preferanse fra min side men tror heller jeg ville hatt left/right content som React.FC istede for JSX.Element her. da ville koden sett en smule pener ut. 

```
<Navbar LeftContent={ProcosysButton} />
```
type NavbarProps = {
    LeftContent?: React.FC;
    midContent?: string;
    RightContent?: React.FC;
    noBorder?: boolean;
};

const Navbar = ({
    LeftContent,
    midContent = '',
    RightContent,
    noBorder = false,
}: NavbarProps): JSX.Element => {
    return (
        <>
            <NavbarWrapper noBorder={noBorder}>
                <LeftContent/>
                <h4>{midContent}</h4>
                <RightContent/>
            </NavbarWrapper>
        </>
    );
};

```
can også kjøre camelCAse på props men da må du caste til  pascalcase

```
const LeftConteren = leftContent;
```